sdk/translation/azure-ai-translation-document/samples/async_samples/sample_begin_translation_async.py

```diff
@@ -59,23 +58,19 @@ async def sample_translation_async():
                     ]
                 )
             ]
-        )  # type: JobStatusResult
+        )
+        result = poller.result()
```

Comment:
i think you have to await this too

Reply:
thanks for catching. Also opened an issue to start running samples in the CI, would have caught that